source/includes/stackpath/_sites.md

```diff
@@ -108,6 +118,11 @@ Attributes | &nbsp;
 `createdAt`<br/>*string* | The date on which the site was created.
 `updatedAt`<br/>*string* | The date on which the site was last updated.
 `services`<br/>*array* | List of services running on the site.
+`edgeAddress`<br/>*string* | The edge address of the site.
+`anycastIp`<br/>*string* | The anycast IP address that domains should be pointed to.
```

Comment:
I see we're using `AnyCast` and `anycast` in our API docs. Can we change them to `Anycast` to match what we have in our UI labels? Thanks!

Reply:
04b37a4